elastic/apm-server
# Record plaintext stack traces as error.stack_trace

## Motivation/summary

Add the `error.stack_trace` field to the `error_logs` data stream, and change our OpenTelemetry processing to record unparsed stack traces as `error.stack_trace` rather than `error.exception.attributes.stacktrace`.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Send a OTLP span with an exception that has a stack trace, using a non-Java SDK
2. Check that an error document is indexed, and the stack trace is recorded in `error.stack_trace`
3. Check that `error.stack_trace` is searchable

## Related issues

Closes #7154
